feature-libs/cart/import-export/components/export-entries/export-entries.service.spec.ts

```diff
@@ -124,6 +113,11 @@ class MockSavedCartDetailsService implements Partial<SavedCartDetailsService> {
     return of({});
   }
 }
+class MockTranslationService {
+  translate(): Observable<string> {
+    return of(columnHeadingTranslation);
```

Comment:
I wanted to have translated content for column headings, so that's why I mocked such method. Otherwise I was getting only translation keys. What's your exact proposal here?